modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,189 @@
+= Use the Control Plane API
+:description: Use the Control Plane API to manage resources in your Redpanda Cloud organization.
+:page-cloud: true
+:page-beta: true
+:tag-clusters: api:ROOT:cloud-api.adoc#tag--ClusterService
+:tag-networks: api:ROOT:cloud-api.adoc#tag--NetworkService
+:tag-operations: api:ROOT:cloud-api.adoc#tag--OperationService
+:tag-resource-groups: api:ROOT:cloud-api.adoc#tag--ResourceGroupService
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your organization's cloud environment outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+For the full Cloud API reference documentation, see xref:api:ROOT:cloud-api.adoc[Redpanda Cloud API Reference].
+
+== Control Plane API
+
+The Control Plane API is one central API that allows you to provision clusters, networks, and resource groups.
+
+The Control Plane API consists of the following endpoints:
+
+* pass:a,m[xref:{tag-clusters}[ClusterService\]]
+* pass:a,m[xref:{tag-networks}[NetworkService\]]
+* pass:a,m[xref:{tag-operations}[OperationService\]]
+* pass:a,m[xref:{tag-resource-groups}[ResourceGroupService\]]
+
+== Long-running operations
+
+Some endpoints do not directly return the resource itself, but instead return an operation. The following is an example response of xref:api:ROOT:cloud-api.adoc#post-/v1beta2/networks[Create Network] (`POST /networks`):
+
+[,bash,role=no-copy]
+----
+"operation": {
+  "id": "cl5nlt1c1scg01plkoe0",
+  "metadata": {
+    "@type": "type.googleapis.com/redpanda.api.controlplane.v1beta2.CreateNetworkMetadata",
+    "network_id": "cl5o0td1dafe70i7460g"
+  },
+  "state": "STATE_IN_PROGRESS",
+  "started_at": "2023-11-08T12:35:01.957Z",
+  "finished_at": null
+}
+----
+
+The response object represents the long-running operation of creating a network. Network creation is an example of an operation that can take a longer period of time to complete.
+
+=== Check operation state
+
+To check the progress of an operation, make a request to the xref:api:ROOT:cloud-api.adoc#get-/v1beta2/operations/-id-[`GET /operations/\{id}`] endpoint using the operation ID as a parameter: 
+
+```bash
+curl https://api.redpanda.com/v1beta2/operations/<operation-id>
+```
+
+The response contains the current state of the operation: `IN_PROGRESS`, `COMPLETED`, or `FAILED`.
+
+== Cluster tiers
+
+When you create a BYOC or Dedicated cluster, you select a usage tier. Each tier provides tested and guaranteed workload configurations for throughput, logical partitions, and connections. Availability depends on the region and the cluster type.
+
+- xref:deploy:deployment-option/cloud/dedicated/dedicated-tiers.adoc[Dedicated tiers and regions]
+- xref:deploy:deployment-option/cloud/byoc-tiers.adoc[BYOC tiers and regions]
```

Comment:
should these now link to your tiers table in the API docs instead?

Reply:
@micheleRP good call, I'll make that change. Currently I am only able to link to an "api-description" anchor in the API reference (and we have kept it to the original GCP vs AWS tables), maybe that can be tweaked later.